client/src/pages/OnBoarding/smallComponents/AvailableHoursButton.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react";
+import { TextField, MenuItem } from "@material-ui/core";
+
+const timeSlots = [];
+for (let i = 0; i <= 24; i++) {
+  for (let j = 0; j <= 60; j = j + 15) {
+    const hour = i < 10 ? "0" + i.toString() : i.toString();
+    const min = j < 10 ? "0" + j.toString() : j.toString();
+    timeSlots.push(hour + min);
```

Comment:
```suggestion
    timeSlots.push(`${hour} : ${min}`);
```
This would save you some time further down